README.md

```diff
@@ -87,17 +87,24 @@ And for all images follow the instructions on [how to publish a change](#4-How-t
 
 If you want to add support for a major postgres major, like `pg-13` you will need to do the steps that are not limited to:
 
-* Generate `postgresql-server-dev-{pg-major}` package with `debbuilder` image. Make sure to add the package index for `pg-major` in `/etc/apt/sources.list.d/pgdg.list`. You can see find the related part [here](https://github.com/citusdata/the-process/blob/master/circleci/images/debbuilder/files/install-builddeps).
+* Generate `postgresql-server-dev-{pg-major}` package with `debbuilder` image. Make sure to add the package index for `pg-major` in `/etc/apt/sources.list.d/pgdg.list`(See [this](https://apt.postgresql.org/pub/repos/apt/dists/)).You can see find the related part [here](https://github.com/citusdata/the-process/blob/master/circleci/images/debbuilder/files/install-builddeps). Basically replace the last one with the new postgres version such as `12` -> `13`.
+
+```bash
+cd debbuilder
+docker build --tag=citus/debbuilder13 . --build-arg PG_MAJORS=13
```

Comment:
To me `--tag=citus/debbuilder:13` would make more sense. It keeps our docker repo clean by not having `N` `debbuilder` images. Instead we will have 1 `debbuilder` image with tags for different versions.

Would this break anything? Assuming we update it throughout

Reply:
I think we can switch to that but the images are already in use so I suggest we do it in another PR and create an issue for it here. (We can do that when building latest 11,12,13 images so that we save some time of building/pushing?)

Reply:
Makes sense.

Now that I am thinking, it might even be better if we tag them with the full postgres version. That will also make it very clear in the circle ci jobs which postgres version we are testing against. This is more so for the final testing images I guess.

Reply:
Yeah it sounds good to have the full postgres version in the tag.

https://github.com/citusdata/the-process/issues/39